src/draw.c

```diff
@@ -250,10 +262,10 @@ static struct dimensions calculate_dimensions(GSList *layouts)
 
 static PangoLayout *layout_create(cairo_t *c)
 {
-        struct screen_info *screen = get_active_screen();
+        const struct screen_info *screen = output->get_active_screen();
 
         PangoContext *context = pango_cairo_create_context(c);
-        pango_cairo_context_set_resolution(context, screen_dpi_get(screen));
+        pango_cairo_context_set_resolution(context, screen->dpi);
```

Comment:
Where is `screen->dpi` initialized? Did you forget to add this to the patch?

Reply:
Yep, I forgot to diff screen.c. Apparently it doesn't break much when you forget to set it. I added it now.